experiment/test_runner.py

```diff
@@ -57,15 +57,13 @@ def test_run_fuzzer_log_file(mocked_communicate, fs):
 EXPERIMENT = 'experiment-name'
 TRIAL_NUM = 1
 FUZZER = 'fuzzer-name-a'
-FULL_FUZZER_NAME = FUZZER
```

Comment:
Marty I had to undo these changes to get `test_integration_runner` to pass.
Do you know if it was passing when you made the change? The test wont run unless you set `TEST_CLOUD_EXPERIMENT_BUCKET` to a bucket.

Reply:
Sorry, forgot to tag @mbarbella-chromium

Reply:
Thanks for the heads up. I didn't set the bucket, so it's very likely that my changes broke it.